src/sidebar/util/sentry.js

```diff
@@ -51,6 +51,8 @@ export function init(config) {
   Sentry.init({
     dsn: config.dsn,
     environment: config.environment,
+    // Do not log Fetch failures to avoid inundating with unhandled fetch exceptions
+    ignoreErrors: ['Fetch operation failed'],
```

Comment:
This string doesn't match the prefix of errors added in `api.js` (the last word here is "failed" but there it is "error").

Reply:
:facepalm: D'oh. I was doing some last-minute string changing...